docs/usage_guide.rst

```diff
@@ -919,6 +1000,30 @@ Finally you can see a piechart of the distribution of your netvalue across all o
    :alt: Distribution of networth by asset
    :align: center
 
+
+Gitcoin Grants
+***************
+
+As a premium user you can also keep track of your Gitcoin grants and generate reports for them.
+
+.. image:: images/gitcoin_grants.png
+   :alt: Gitcoin grant events
+   :align: center
+
+You can type grant id, select a period you are interested in and press the fetch button to retrieve the events for this
```

Comment:
Here we should mention how to get the grant id. In my case I go to the project page and in the url `https://gitcoin.co/grants/149/rotki` the grant id is before the project name and after grants. I don't know if there is a better way to find it

Reply:
```suggestion
You need to first find the id ofthe grant you are interested in. When you visit the grant, the id is what comes after the ``grants/`` part of the url. For example for rotki (https://gitcoin.co/grants/149/rotki) the id is ``149``.

 You type the grant id, select a period you are interested in and press the fetch button to retrieve the events for this
```